Samples/SampleApp.Tests/DomainTests.cs

```diff
@@ -15,7 +14,7 @@ public class DomainTests
         public DomainTests()
         {
             domainEntities = Types.InAssemblyOf<DomainClass>("Domain Entities", 
-                types => types.Where(t=>t.Namespace.StartsWith("SampleApp.Domain")));
+                type => type.Namespace.StartsWith("SampleApp.Domain"));
```

Comment:
Not sure if this is an issue, but this would stop us doing a select many. Can you think of a reason you would need to project from a type into multiple types?